users/app.py

```diff
@@ -78,6 +78,36 @@ def upsert_user_in_db(user_object, users_collection):
         upsert=True).upserted_id
 
 
+def update_user_authorization_in_db(user_id, is_organizer, users_collection):
+    """Updates the authorization of the given user in the database.
+
+    Assumes caller has authorization to make this change in the database.
+    I.e. caller route should first make sure request maker was authorized
+    to update the given user.
+
+    Args:
+        user_id (str): The id of the user to change.
+        is_organizer (bool): The authorization value to set.
+
+    Returns:
+        ObjectID: The ID of the updated object from the db. This can be used
+            to find the object with `collection.find_one(object_id)`.
+
+    Raises:
+        KeyError: Bad `user_id`/user not found in db.
+        TypeError: `is_organizer` is not a bool.
+    """
+    if not isinstance(is_organizer, bool):
+        raise TypeError("Trying to set authorization to a non-bool type.")
```

Comment:
Question: is this a good way to handle this? This helper function should only be used by 1 route in the same file later, so it probably will never be called incorrectly, so maybe this case isn't even needed. I wanted to type check `is_organizer` though so that it could never be entered as a non-bool value into the database. Would [PEP 484](https://www.python.org/dev/peps/pep-0484/) (type hints) or [PEP 3107](https://www.python.org/dev/peps/pep-3107/) (function annotations) be a better way to just hint at the correct type without strict code enforcement, or is this fine?

Reply:
PEP 484 and duck typing (e.g. `bool(is_organizer)`) should be sufficient here.

Overall, I'd recommend PEP 484 if you have time.

Reply:
Done https://github.com/knative-portability/large-events/pull/11/commits/a94e699754b72786210719bad71125fdc9b94f24 . Does that look right?

Reply:
I think so -- you might want to check and see if you can get the type checking to alarm on passing the string `"False"` to `is_organizer`.